addons/binding/org.openhab.binding.homematic/ESH-INF/thing/bridge.xml

```diff
@@ -22,6 +22,7 @@
 				<options>
 					<option value="auto">Auto</option>
 					<option value="ccu">CCU</option>
+                    <option value="noccu">NoCCU</option>
```

Comment:
Github shows some troubles in this markup, I would guess spaces and tabs are combined, can you (auto)format the file.